go-fuzz/main.go

```diff
@@ -29,10 +34,13 @@ var (
 	flagCoverCounters = flag.Bool("covercounters", true, "use coverage hit counters")
 	flagSonar         = flag.Bool("sonar", true, "use sonar hints")
 	flagV             = flag.Int("v", 0, "verbosity level")
+	flagMasterStats   = flag.String("masterstats", "", "masterstats HTTP server listen address")
```

Comment:
please call the flag just "http"
I want to expose more stats via http, including hub/slave stats. Http is also shorter and used by godoc and go tool trace.
But the check that this flag must not be provided on slave is correct -- all stats should be aggregated and exposed by master.